Gemfile

```diff
@@ -6,7 +6,8 @@ ruby File.read(".ruby-version").strip
 gem 'rails', '~> 6.1.7.1'
 
 # Use mysql as the database for Active Record
-gem 'mysql2', '~> 0.5.4'
+
+gem 'sqlite3', '~> 1.3', '>= 1.3.11'
 
```

Comment:
This should be added as a development dependency only to prevent it from being installed on the main server.